database/src/main/resources/migrations/ddl/V1.0.0__create_users_sessions.sql

```diff
@@ -1,22 +1,25 @@
 CREATE TABLE users (
-                       id INTEGER GENERATED ALWAYS AS IDENTITY,
-                       email VARCHAR(128) UNIQUE NOT NULL,
-                       first_name VARCHAR(128) NOT NULL,
-                       last_name VARCHAR(128) NOT NULL,
-                       verified BOOLEAN NOT NULL DEFAULT FALSE,
-                       hashed_password VARCHAR(128) NOT NULL,
-                       verification_token VARCHAR(128) NOT NULL,
-                       PRIMARY KEY (id)
+       id INTEGER GENERATED ALWAYS AS IDENTITY,
```

Comment:
We shouldn't edit existing scripts because the database is migrated through flyway
Instead, a new script should be created in which we ALTER TABLE